### ISSUE TITLE
evaluation on the other dataset of udacity

### ISSUE THREAD

**Issue by daothanh2011** [uzh-rpg/rpg_public_dronet#47]
Thank you for your great work.
Currently, I am working based on your work. However, when I validated the model by using HMB_2 dataset (since it's quite difficult to download the HMB_3 dataset), the result shows not good as the paper.
I used script data_preprocessing to pre-processing the udacity dataset, and copy to the validation folder.

This is the output of the validation:
EVA = -0.5063827037811279
RMSE = 0.3688988983631134
Written file /Autonomous_drone/rpg_public_dronet/model/dronet_model/dronet/test_regression.json
EVA = -1.0062613594628216
RMSE = 0.42530794406649747
Written file /Autonomous_drone/rpg_public_dronet/model/dronet_model/dronet/random_regression.json
EVA = 0.0
RMSE = 0.30026811361312866
Written file /Autonomous_drone/rpg_public_dronet/model/dronet_model/dronet/constant_regression.json
Written file /Autonomous_drone/rpg_public_dronet/model/dronet_model/dronet/predicted_and_real_steerings.json
Average accuracy =  0.8614232209737828
Precision =  0.10869565217391304
Recall =  0.10869565217391304
F1-score =  0.11904761904761904
Written file /Autonomous_drone/rpg_public_dronet/model/dronet_model/dronet/test_classification.json
Average accuracy =  0.5056179775280899
Precision =  0.07196969696969698
Recall =  0.07196969696969698
F1-score =  0.12582781456953643

Could you give me any advice to improve the accuracy?
Thank you in advance!

**Comment by antonilo** [comment]
Could you maybe double check that the data and pipeline is alright? A possibility would be to overfit to that dataset. Then, maybe analyze what are the places with higher error and train on those cases: a form of hard mining. This would drastically improve performance! Hope it helps!
